src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -161,23 +160,21 @@ public void setup() throws Exception {
             testConfig.getSourceStorageAccountName(),
             null,
             retryOptions);
+    snapshotIds = new ArrayList<>();
   }
 
   @After
   public void teardown() throws Exception {
     dataRepoFixtures.resetConfig(steward);
-    if (snapshotId != null) {
-      dataRepoFixtures.deleteSnapshot(steward, snapshotId);
-      snapshotId = null;
-    }
-    if (snapshotByRowId != null) {
-      dataRepoFixtures.deleteSnapshot(steward, snapshotByRowId);
-      snapshotId = null;
-    }
-    if (snapshotByAssetId != null) {
-      dataRepoFixtures.deleteSnapshot(steward, snapshotByAssetId);
-      snapshotId = null;
-    }
+    snapshotIds.forEach(
```

Comment:
Would it make sense to just call `clearEnvironment` here?

Reply:
I haven't wired up the azure integration test to use the synapseUtils methods. I think I would actually need to write up something different since synapseUtils is for our connected tests.